test/Microsoft.AspNetCore.Mvc.FunctionalTests/DispatchingTests.cs

```diff
@@ -0,0 +1,1259 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Routing;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.FunctionalTests
+{
+    public class DispatchingTests : IClassFixture<MvcTestFixture<DispatchingWebSite.Startup>>
```

Comment:
Did you copy-paste these from the routing tests? I'm going to assume that's the case and skip it for now. let me know if I'm wrong.

Reply:
It is copy paste apart from setting up the necessary middleware.